buildSrc/src/main/java/com/uber/okbuck/composer/jvm/JvmBuckRuleComposer.java

```diff
@@ -21,6 +21,10 @@ public static String test(JvmTarget target) {
     return "test_" + target.getName();
   }
 
+  public static String integrationTest(JvmTarget target) {
```

Comment:
The name of this method can be made better? What does this string represents? is that the target name? If so it could be `getIntegrationTestTargetName`

Reply:
This follow the conventions for all other methods (such as `src(`, `bin(` and `test(`. Refactoring it it out of the scope of this PR